AirCasting/CreateSessionViews/ConfirmCreatingSessionView.swift

```diff
@@ -118,7 +123,26 @@ struct ConfirmCreatingSessionView: View {
             })
                 .buttonStyle(BlueButtonStyle())
         }
-            .padding()
+        .padding()
+    }
+
+    func getAndSaveStartingLocation() {
+        if sessionContext.sessionType == .fixed {
```

Comment:
let's split this block into some more private functions for readability

edit: I see it's just been moved from somewhere, disregard ;)